### DISCUSSION TITLE
Is there an official response: Which is better, app router vs page router? Is there accurate data to support the explanation?

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#68619] [Help]
### Summary

Is there an official response: Which is better, app router vs page router? Is there accurate data to support the explanation?  We need to refactor the website, and we are hesitating whether to use an app or a page router for better performance.

### Additional information

```js
Is there an official response: Which is better, app router vs page router? Is there accurate data to support the explanation?  We need to refactor the website, and we are hesitating whether to use an app or a page router for better performance.
```

**Comment by parlay96** [comment]
@Timer Hello, can you answer? thank you

**Comment by Juneezee** [comment]
This has already been answered already. See the response here: https://github.com/vercel/next.js/discussions/56655#discussioncomment-8791401

If you plan to use the latest React features such as Server Components, Server Actions, etc., then you have to use App Router
